src/components/Admin/Upload.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from "react";
+import { RiAttachment2 } from "react-icons/ri";
+import { FaTimes } from "react-icons/fa";
+import { toast } from "react-hot-toast";
+import { Row, Col } from "react-bootstrap";
+
+const Upload = ({ text, setObjects }) => {
+  const [files, setFiles] = useState([]);
+  const [uploading, setUploading] = useState(false);
+
+  const handleInput = (e) => {
+    setUploading(true);
+    if (files.length + e.target.files.length > 5) {
+      toast("❌ Exceeds 5 files!");
+      return;
+    }
+    const fileList = [];
+    Object.entries(e.target.files).forEach(([ind, file]) => {
+      if (file.size > 1048576) {
```

Comment:
can you make this a constant at the top of the code like 

const 1MB = 1048576